conductr_cli/sandbox_stop.py

```diff
@@ -1,13 +1,7 @@
-from conductr_cli import sandbox_common, terminal
-from conductr_cli.screen_utils import headline
-import logging
+from conductr_cli import sandbox_stop_docker
 
 
 def stop(args):
     """`sandbox stop` command"""
 
-    log = logging.getLogger(__name__)
-    running_containers = sandbox_common.resolve_running_docker_containers()
-    if running_containers:
-        log.info(headline('Stopping ConductR'))
-        terminal.docker_rm(running_containers)
+    sandbox_stop_docker.stop(args)
```

Comment:
Also, the check if JVM or Docker version should be used based on the ConductR version is missing here, e.g. in case someone is using `sandbox stop` from the terminal window.